maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -165,6 +168,15 @@ class Orchestra(
         return true
     }
 
+    @Synchronized
+    private fun initJsEngine(config: MaestroConfig?) {
+        if (this::jsEngine.isInitialized) {
+            jsEngine.close()
+        }
+        val shouldUseGraalJs = config?.ext?.get("jsEngine") == "graaljs" || System.getenv("MAESTRO_USE_GRAALJS") == "true"
+        jsEngine = if (shouldUseGraalJs) GraalJsEngine() else RhinoJsEngine()
```

Comment:
Note, we are now creating a new instance of the JsEngine on every flow execution. Before we were caching the JsEngine instance. This shouldn't have any significant impact on performance.